compose/cli/main.py

```diff
@@ -710,7 +712,9 @@ def ps(self, options):
         Options:
             -q, --quiet          Only display IDs
             --services           Display services
-            --filter KEY=VAL     Filter services by a property
+            --filter KEY=VAL     Filter s        # TODO add support for "-" as path
+
+ervices by a property
```

Comment:
Didn't understand this part... Is it intentional?

Reply:
bad copy/paste